pkgs/development/compilers/glslang/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "glslang-git-${version}";
-  version = "2018-06-21";
+  version = "2018-08-16";
 
   src = fetchFromGitHub {
-    owner = "KhronosGroup";
+    owner = "lenny-lunarg";
```

Comment:
Why change the source to this fork?

Reply:
Upstream forked to backport an important bugfix. See https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/252 for details. The source will be reverted in the next update.

Reply:
Looks that way. Are you thinking incorporating the patch nix-side would be better?

Reply:
You can use the upstream url and `fetchpatch` then.